### PR TITLE
feat: add docker compose config

### PR DESCRIPTION
**Pull request by leechor**

### CODE REVIEW

File: script/bin/auto.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ tips() {
 start() {
   pid=$(cat ${PID_PATH}/${PID_FILE})
   if [ -z $pid ]; then
-    nohup java -Ddruid.mysql.usePingMethod=false -Xms512M -Xmx2048M -XX:PermSize=512M -XX:MaxPermSize=1024M -XX:+HeapDumpOnOutOfMemoryError -Xverify:none -cp ${CLASS_PATH} org.dinky.Dinky >/dev/null 2>&1 &
+    nohup java -Ddruid.mysql.usePingMethod=false -Xms512M -Xmx2048M -XX:PermSize=512M -XX:MaxPermSize=1024M -XX:+HeapDumpOnOutOfMemoryError -Xverify:none -cp ${CLASS_PATH} org.dinky.Dinky  &
```
Comment (leechor) [inline script/bin/auto.sh]: to docker log

**Review by aiwenmo — APPROVED** [review]
LGTM

File: script/bin/auto.sh [hunk 2]
```diff
@@ -52,9 +52,10 @@ start() {
...
   pid=$(cat ${PID_PATH}/${PID_FILE})
   if [ -z $pid ]; then
-    nohup java -Ddruid.mysql.usePingMethod=false -Xms512M -Xmx2048M -XX:PermSize=512M -XX:MaxPermSize=1024M -XX:+HeapDumpOnOutOfMemoryError -Xverify:none ${JMX} -cp ${CLASS_PATH} org.dinky.Dinky >/dev/null 2>&1 &
+    nohup java -Ddruid.mysql.usePingMethod=false -Xms512M -Xmx2048M -XX:PermSize=512M -XX:MaxPermSize=1024M -XX:+HeapDumpOnOutOfMemoryError -Xverify:none ${JMX} -cp ${CLASS_PATH} org.dinky.Dinky &
     echo $! >${PID_PATH}/${PID_FILE}
-    echo "........................................Start Dlink Successfully........................................"
+    echo "........................................Start Dlink with Jmx Successfully.....................................
+    ..."
   else
     echo "Dlink pid $pid is in ${PID_PATH}/${PID_FILE}, Please stop first !!!"
```
Comment (Zzm0809) [inline script/bin/auto.sh]: change to Dinky

**Review by aiwenmo — APPROVED** [review]
LGTM

**Pull request merged**